docs/accessibility/policies_faq.md

```diff
@@ -0,0 +1,41 @@
+---
+permalink: /accessibility/policies_faq/
+title: Accessibility policies and frequently asked questions
+sidenav:
+  - text: Decorative images
+    href: "#using-decorative-images"
+---
+[\<\< Back to Login.gov Accessibility home]({% link accessibility.md %})
```

Comment:
Did the designs use two <'s? What about using the &laquo;?

Reply:
The designs did, but it might make more sense to use « or ← or even none. My gut says arrow since « is a [type of quotation mark](https://en.wikipedia.org/wiki/Guillemet)?

Reply:
Works for me!